src/signals/incident/containers/IncidentReplyContainer/IncidentReplyContainer.tsx

```diff
@@ -159,7 +159,11 @@ const IncidentReplyContainer = () => {
   }, [session, getQuestionnaire, getIncident])
 
   const formattedDate = useMemo(
-    () => (incident ? formatDate(new Date(incident.created_at)) : ''),
+    () =>
+      (incident ? formatDate(new Date(incident.created_at)) : '').replace(
+        ':',
+        '.'
+      ),
```

Comment:
Assuming we want the dot notation everywhere. This should be handled in the util, not here. Else you need to everywhere the util is used. 
src/signals/incident/containers/IncidentReplyContainer/utils.ts

Reply:
Done as you said.